src/main/resources/com/mathworks/ci/RunMatlabCommandBuilder/help-matlabCommand.html

```diff
@@ -0,0 +1,10 @@
+<div>
+ Enter the command you want to run in MATLAB. If the command represents a MATLAB function or script, do not specify the file extension.<br>
+ <b>Examples:</b><br>
+ <b>Run Commands:</b> results = runtests(‘IncludingSubfolders’,true); assert(all(~[results.Failed]))<br>
+ <b>Run a script:</b> runMyScript<br>
```

Comment:
Use consistent capitalization.

Suggest just simplifying this to:
```
Example: myscript
Example: results = runtests(‘IncludingSubfolders’,true); assert(all(~[results.Failed]))
```

Reply:
Changed to following. I think being explicit about examples for command and script is good idea  
Run command : results = runtests(‘IncludingSubfolders’,true); assert(all(~[results.Failed]))
Run a script : runMyScript